examples/ruby-on-rails/Earthfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:2.6.5
+WORKDIR /ruby-on-rails-example
+ENV BUNDLER_VERSION=2.1.4
+
+deps:
+    RUN curl -sL https://deb.nodesource.com/setup_12.x  | bash - && \
+      apt-get install -y nodejs
+    RUN gem install bundler:2.1.4
+    COPY Gemfile Gemfile.lock ./
+    RUN bundle config build.nokogiri --use-system-libraries
+    RUN bundle check || bundle install
+    SAVE ARTIFACT Gemfile.lock AS LOCAL ./Gemfile.lock
+    SAVE IMAGE
+build:
```

Comment:
Mind adding a blank line between targets? It helps keep the examples consistent in style.

Reply:
done